caom2pipe/data_source_composable.py

```diff
@@ -382,8 +382,14 @@ def _append_work(self, prev_exec_time, exec_time, entry_path):
                     # send the dir_listing value
                     if self.default_filter(entry):
                         entry_stats = entry.stat()
-                        if exec_time >= entry_stats.st_mtime >= prev_exec_time:
-                            self._temp[entry_stats.st_mtime].append(entry.path)
+                        if (
+                            exec_time
+                            >= entry_stats.st_mtime
+                            >= prev_exec_time
+                        ):
```

Comment:
This format in this file is funny and I can't tell if it's because of the width of the line or your IDE.

Reply:
IDEA recommends `a >= x >= b` as the format. If you have a different preference, let me know and I'll change it, and ignore IDEA's notifications.

Reply:
It's not the expression but how it's broken into rows.

Reply:
120 wide rows will fix this layout.